tests/queries/0_stateless/02494_query_cache_case_agnostic_matching.sql

```diff
@@ -1,5 +1,6 @@
--- Tags: no-parallel
+-- Tags: no-parallel, no-random-settings
```

Comment:
You disable all random settings for this test, maybe set the query cache settings to some value explicitly at the start of the tests?

Reply:
Good idea, done.